CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.0.0b1
+- Update python-base submodule: config loader now raises exception on duplicated name in kubeconfig [kubernetes-client/python-base#47](https://github.com/kubernetes-client/python-base/pull/47)
```

Comment:
I don't think we want "Update python-base submodule:". That is an implementation detail and users don't care about it.

Reply:
Updated